lib/dispatch-rider/notification_services/aws_sns.rb

```diff
@@ -1,3 +1,4 @@
+require 'retries'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.